amarillo/models/Carpool.py

```diff
@@ -110,7 +119,83 @@ class Region(BaseModel):
     bbox: Tuple[NumType, NumType, NumType, NumType] = Field(
         description="Bounding box of the region. Format is [minLon, minLat, maxLon, maxLat]",
         examples=[[10.5,49.2,11.3,51.3]])
+    
+class RidesharingInfo(BaseModel):
+    number_free_seats: int = Field(
+        description="Number of free seats",
+        ge=0,
+        examples=[3])
+    
+    same_gender: Optional[YesNoEnum] = Field(
+        None, 
+        description="Trip only for same gender:"
+                    "1: Yes"
+                    "2: No",
```

Comment:
Does this make sense? How would I know which gender?

Reply:
I'd leave it as it is for now. In Mitanand we will change this in the GRFS spec. And Note to myself, to update this accordingly then.